scss/utilities/mixins/organisms/_footer.scss

```diff
@@ -0,0 +1,36 @@
+@charset "UTF-8";
+
+//
+// Footer Mixins
+//
+// Mixins for the global footer.
+//
+// Style guide: Mixins.Organisms.Footer
+//
+
+@mixin footer-colors {
+  background-color: $color-cardinal-red;
+  color: $color-white;
+
+  a {
+    color: rgba($color-white, .9);
+    text-decoration: none;
+
+    &:hover {
+      color: $color-white;
+      text-decoration: underline;
+    }
+  }
+}
+
+@mixin footer-wrapper {
+  @include footer-colors;
+  display: flex;
+  padding: 2.5rem;
+}
+
+@mixin footer-copyright {
+  @include footer-colors;
```

Comment:
1. The a tags in this mixin are not needed as it is taken care of in the footer-wrapper code and are redundant. 
2. This mixin is probably not needed and could be rolled into another
3. This mixin would be best refactored in to a common copyright mixin and/or component rather than be specific to the footer.
4. No code comments! We have to document all of our mixins.
5. This should be in its own file.